completion/bash/_phpbrew

```diff
@@ -6660,6 +6660,270 @@ local argument_min_length=0
         fi
     fi
 };
+___phpbrew_complete_system ()
+{
+local comp_prefix=___phpbrew
```

Comment:
The indent here looks a bit weird

Reply:
This code is autogenerated. See `Makefile`.

Reply:
Ah I see